secator/query/_base.py

```
@@ -7,67 +7,66 @@
-    @abstractmethod
-    def _execute_count(self, query: dict) -> int:
-        """Backend-specific count implementation."""
-        pass
+	"""Abstract base class for query backends."""
+
+	name: str = 'base'
+
+	PROTECTED_FIELDS = [
+		'_context.workspace_id',
+		'_context.workspace_duplicate',
+	]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)